src/server.c

```diff
@@ -53,11 +53,25 @@ int send_response(int fd, char *header, char *content_type, void *body, int cont
     const int max_response_size = 262144;
     char response[max_response_size];
 
-    // Build HTTP response and store it in response
+    int header_length = strlen(header);
+    int content_length = strlen(body);
```

Comment:
Content_length is passed into the function, you don't necessarily have to redefine it here, not that this way can't work mind you.